File: web-ui/src/displays/components/Heading.js
import React from "react";
import { withStyles } from '@material-ui/core/styles';
import AbstractHeading from "../../../gen/displays/components/AbstractHeading";
import HeadingNotifier from "../../../gen/displays/notifiers/HeadingNotifier";
import HeadingRequester from "../../../gen/displays/requesters/HeadingRequester";
import DisplayFactory from 'alexandria-ui-elements/src/displays/DisplayFactory';

const styles = theme => ({});

class Heading extends AbstractHeading {

	constructor(props) {
		super(props);
		this.notifier = new HeadingNotifier(this);
		this.requester = new HeadingRequester(this);
	};


}

export default withStyles(styles, { withTheme: true })(Heading);
DisplayFactory.register("Heading", withStyles(styles, { withTheme: true })(Heading));

File: framework/ui/src/io/intino/alexandria/ui/displays/components/Heading.java
package io.intino.alexandria.ui.displays.components;

import io.intino.alexandria.core.Box;
import io.intino.alexandria.ui.displays.notifiers.HeadingNotifier;

public class Heading<DN extends HeadingNotifier, B extends Box> extends AbstractHeading<B> {

    public Heading(B box) {
        super(box);
    }

}